acquia/acquia_cms
# ACMS-747: Changed the image display size option to Actual size instead of responsive.

**Motivation**
* The image display size option is set to "Responsive" which is not letting the image style alter the image on UI for a site visitor.
Fixes #747

**Proposed changes**
* Changed the image display size option to "Actual Size" which helps in using the image size selected and is also responsive. (See attached screenshot) 

**Alternatives considered**
* NONE

**Testing steps**
* To test, create a new content of type page.
* Select the image component in the layout canvas field.
* Double click to add image and upload the image.
* Select image style from image style dropdown field.
* Save and visit the page to see the changes.

**Merge requirements**

**Screenshot**
![Screenshot 2021-05-12 at 6 16 13 PM](https://user-images.githubusercontent.com/15887127/117977161-2a187900-b34e-11eb-861b-521d6cdbcca5.png)
